src/main/scala/com/cognite/sdk/scala/v1/resources/raw.scala

```diff
@@ -102,15 +105,24 @@ class RawRows[F[_]](val requestSession: RequestSession[F], database: String, tab
     extends WithRequestSession[F]
     with Readable[RawRow, F]
     with Create[RawRow, RawRow, F]
-    with DeleteByIds[F, String] {
+    with DeleteByIds[F, String]
+    with PartitionedFilterF[RawRow, RawRowFilter, F] {
+  implicit val errorOrUnitDecoder: Decoder[Either[CdpApiError, Unit]] =
+    EitherDecoder.eitherDecoder[CdpApiError, Unit]
+
+  implicit val stringItemsDecoder: Decoder[Items[String]] = deriveDecoder[Items[String]]
+
+  implicit val errorOrStringItemsDecoder: Decoder[Either[CdpApiError, Items[String]]] =
+    EitherDecoder.eitherDecoder[CdpApiError, Items[String]]
+
   import RawRows._
   override val baseUri =
     uri"${requestSession.baseUri}/raw/dbs/$database/tables/$table/rows"
 
+  val cursorsUri = uri"${requestSession.baseUri}/raw/dbs/$database/tables/$table/cursors"
+
   // raw does not return the created rows in the response, so we'll always return an empty sequence.
-  override def createItems(items: Items[RawRow]): F[Seq[RawRow]] = {
-    implicit val errorOrUnitDecoder: Decoder[Either[CdpApiError, Unit]] =
-      EitherDecoder.eitherDecoder[CdpApiError, Unit]
+  override def createItems(items: Items[RawRow]): F[Seq[RawRow]] =
```

Comment:
There's a good chance the `errorOrUnitDecoder` there is actually needed. :) Make sure you test it properly before you remove it, maybe we can add a test with an invalid call to `createItems`

Reply:
It was moved one scope up, but I can move it back again.

Reply:
I moved it back